dists/install.sh

```diff
@@ -58,14 +54,14 @@ main() {
     echo
   }
 
-  if [ "x$DEBUG" = "xtrue" ] ; then
+  if [ -n "$DEBUG" ] ; then
```

Comment:
it's not equivalent. Why don't we keep the equivalent?
```suggestion
  if [ "$DEBUG" = "true" ] ; then
```

Reply:
It's not equivalent, I change it to be more consistent with the usage of the `DEBUG` of the CLI itself (which can be used with `DEBUG=1`).